### ISSUE TITLE
[FEATURE] Add "net6.0-windows" support and not dependent "SkiaSharp.NativeAssets.macOS"

### ISSUE THREAD

**Issue by SlimeNull** [mono/SkiaSharp#2236]
![image](https://user-images.githubusercontent.com/69663231/188142829-2ab73fa6-8048-432b-a8a2-1f86f4b66129.png)

We do not need "SkiaSharp.NativeAssets.macOS" on Windows, so please add "net6.0-windows" target support and just use native assets of win32.

**Comment by dongyi-cai-windsab** [comment]
Agree, all `SkiaSharp.NativeAssets.xxx` should depend on on SkiaSharp, not the other way around. 

Same to .NETStandard 2.0 and lower.

**Comment by mattleibow** [comment]
The reason we have the plain net6.0 depend on macos and windows is that you can publish a net6.0 console app that can run on both mac and windows.

However, the OP issue is a good suggestion. We have a netX.X-windows TFM, so we know that only runs on windows - so we only need the windows assets. I'll set that one up. Thanks!
